eng/cake/dotnet.cake

```diff
@@ -142,12 +143,17 @@ Task("dotnet-samples")
     .Does(() =>
     {
         var tempDir = PrepareSeparateBuildContext("samplesTest");
+        var properties = new Dictionary<string, string>();
 
-        RunMSBuildWithDotNet("./Microsoft.Maui.Samples.slnf", new Dictionary<string, string> {
-            ["UseWorkload"] = "true",
-            // ["GenerateAppxPackageOnBuild"] = "true",
-            ["RestoreConfigFile"] = tempDir.CombineWithFilePath("NuGet.config").FullPath,
-        }, binlogPrefix: "sample-");
+        if(useNuget)
```

Comment:
Need to figure if we can remove this

Reply:
i think we want to might want to  create a copy of the sandbox app for starting adding the UITests , and we don't need to mess with this.